bootstrap/http/compose-dev.yaml

```
@@ -16,6 +16,7 @@ services:
     environment:
       MYSQL_DATABASE: weave
       MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
+    command: --default-time-zone='+09:00'
```

p3) TZ enviroment로 되지 않나요?!
```suggestion
      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
      TZ=Asia/Seoul
```

---

그게더 직관적이겠네용